pyproject.toml

```diff
@@ -243,27 +243,6 @@ zip-safe = false
 namespaces = false
 
 
-[tool.black]
-line-length = 88
-target-version = ['py39']
-include = '\.pyi?$'
-exclude = '''
-/(
-    \.eggs
-  | \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | _build
-  | buck-out
-  | build
-  | dist
-  | yt/frontends/stream/sample_data
-)/
-| yt/visualization/_colormap_data.py
-'''
```

Comment:
Can we remove the black-specific configuration from `pyproject.toml` since it is still in use by blacken-docs?

Reply:
Good point. I (mostly) reverted it, but I took the opportunity to clean up unnecessary parameters.